apiserver/spaces/spaces.go

```diff
@@ -102,3 +104,19 @@ func (api *spacesAPI) ListSpaces() (results params.ListSpacesResults, err error)
 	}
 	return results, nil
 }
+
+// RefreshSpaces refreshes spaces from substrate
+func (api *spacesAPI) ReloadSpaces() error {
+	canRead, err := api.authorizer.HasPermission(permission.ReadAccess, api.backing.ModelTag())
```

Comment:
I don't think Read permission is enough to invoke ReloadSpaces. I would expect Write level to mutate database objects.

We also need the APIServer test that tries with different permissions and ensures that we have the right perm set. (We should try as a readonly user/no perms and show that we get denied, but it succeeds with Write or Admin).